instructure/canvas-android
# [Student][MBL-13745] Generate pairing code

To test:
The permission check only works on beta right now (USERS-473)
* Should be able to use the pairing code generated to pair a student in the parent app.
* Should be able to scan the qr code and see the pairing code as a query param using a third party qr scanner (or it will open the parent app to the dashboard, just depends on the qr scanner you use)
* Testing without network connection should prevent navigating to this screen from settings
* Testing as a teacher/observer using the student app should show an error on this screen (they can't generate pairing codes, even though the permission isn't restricted)
